nemo_skills/pipeline/utils.py

```diff
@@ -762,7 +776,8 @@ def add_task(
     executors = []
     # assuming server always has the largest resources request, so it needs to go first
     if server_config is not None:
-        server_cmd, num_server_tasks = get_server_command(**server_config, cluster_config=cluster_config)
+        server_port = server_config["server_port"]
```

Comment:
Sorry, I just realized that in this case we can completely remove this line and let it come from server_config directly via **. This will also make error message more descriptive in case it's missing

Reply:
added in [dc1c3b2](https://github.com/NVIDIA/NeMo-Skills/pull/298/commits/dc1c3b2719c7e4f9471631ca21a3273f6fe77c41)